manage.py

```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'StuDash.settings')
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'studash_.settings')
```

Comment:
What is the reason for the "_"?
If it will take some time then leave it like @ifireball suggested

Reply:
It's the name of the new directory. I tried to name it just 'studash' but Git didn't recognized the name change, so that's why I added the '_'.

Reply:
strange,
I renamed it, and git detected the change.
Also, you can use 'git mv'
Are you using windows OS?

Anyhow, it's not that important, and you don't need to spend time on that.
Having a module name that ends with "_" is unconventional same as using capital letters.